src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -171,7 +171,7 @@ protected async Task VerifySendEnterThroughToEnterAsync(string initialMarkup, st
 
             var service = GetCompletionService(document.Project);
             var completionList = await GetCompletionListAsync(service, document, position, RoslynTrigger.Invoke);
-            var item = completionList.Items.First(i => (i.DisplayText + i.DisplayTextSuffix).StartsWith(textTypedSoFar));
```

Comment:
should we add this API the banned list internally, so we're forces to use ItemsInternal everywhere?

Reply:
That's a good idea. Done!
<img width="595" alt="image" src="https://user-images.githubusercontent.com/788783/172267642-428a6381-ef04-4667-b2bf-8f2f17349d63.png">